modules/installation-cloudformation-dns.adoc

```diff
@@ -9,6 +9,16 @@
 You can use the following CloudFormation template to deploy the networking
 objects and load balancers that you need for your {product-title} cluster.
 
+[IMPORTANT]
+====
+If you are deploying the networking objects and load balancers, you must ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For example:
+[source,yaml]
+----
+RegisterPublicSubnetTags: http://kubernetes.io/role/elb
+RegisterPrivateSubnetTags: http://kubernetes.io/role/internal-elb
```

Comment:
1. this format is not good for users,  they may think the `RegisterPublicSubnetTags` is the key, and `http://kubernetes.io/role/internal-elb` is the value, all things present as yaml format would be considered  as _code_ (tag key/value), not  description (_RegisterPublicSubnetTags_)  
2. the key is `kubernetes.io/role/internal-elb`, value is _[empty]_

Reply:
Tes, the tag format is a key-value pair, here the key should be `kubernetes.io/role/internal-elb` and no `http://`.